cognite_toolkit/_builtin_modules/sourcesystem/sap/auth/transformation.Group.yaml

```diff
@@ -0,0 +1,47 @@
+name: gp_transformations_{{ sourceId }}
+sourceId: {{ transformationServicePrincipalGroupObjectId }}
+metadata:
+  origin: cognite-toolkit
+capabilities:
+  - rawAcl:
+      actions:
+        - READ
+      scope:
+        tableScope:
+          dbsToTables:
+            {{ rawDb }}: {}
+  - dataModelInstancesAcl:
+      actions:
+        - READ
+        - WRITE
+        - WRITE_PROPERTIES
+      scope:
+        spaceIdScope:
+          spaceIds:
+            - {{ instanceSpace }}
+  - dataModelsAcl:
+      actions:
+        - READ
+      scope:
+        spaceIdScope:
+          spaceIds:
+            - {{ schemaSpace }}
+  # Needed to create a nonce when creating the transformation
```

Comment:
I suggest to move this to the top with a comment like 

```
# minimum runtime capabilities
...
```

then 

```
# transformation-specific capabilities 
....

```

to make it clearer that they always need to be present for non-human users